### PR TITLE
MM-T645 added test for deleting incoming webhook

### PR DESCRIPTION
**Pull request by JeremyShih**
#### Summary

Delete Incoming Webhook

#### Ticket Link
[MM-T645](https://automation-test-cases.vercel.app/test/MM-T645)
mattermost/mattermost-server#15254

### CODE REVIEW

File: e2e/cypress/integration/integrations/incoming_webhook/delete_incoming_webhook_spec.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
```
Comment (saturninoabril) [inline e2e/cypress/integration/integrations/incoming_webhook/delete_incoming_webhook_spec.js:10]: Please remove. We'll add this in the future once we found this test to be stable.
```suggestion
```

**Review by saturninoabril — APPROVED** [review]
Thanks @JeremyShih, LGTM! Tested and passed.

**Review by prapti — APPROVED** [review]
Great work @JeremyShih! Thank you for the contribution 🎉

**Pull request merged**